src/components/button/button.stories.tsx

```diff
@@ -12,7 +12,7 @@ import { disableStorybookControls } from '../../../.storybook/utils';
 import { EuiButton, Props as EuiButtonProps } from './button';
 
 const meta: Meta<EuiButtonProps> = {
-  title: 'EuiButton',
+  title: 'Navigation/EuiButton/EuiButton',
```

Comment:
I'd actually leave the button components separate / as their own components. The distinction is not a very clear one but to me it's about parent/child relationships vs sibling relationships: `EuiButton` is a distinct separate (but sibling) component to `EuiButtonEmpty` and they can be used apart from one another, whereas `EuiFlyoutBody` is a subcomponent / child of `EuiFlyout` and does not make sense outside of `EuiFlyout`.

Let me know if I'm way off in this thinking!

```suggestion
  title: 'Navigation/EuiButton',
```